translated/pt_br/players/faq.md

```diff
@@ -7,25 +7,23 @@ description: Dúvidas mais frequentes dos jogadores e administradores de servido
 
 Há muitas perguntas feitas com frequência, então compilamos uma lista delas aqui.
 
-## Perguntas Gerais
-
-### Quais versões do Minecraft o Fabric é compatível?
+## Quais versões do Minecraft são compatíveis com o Fabric? {#what-minecraft-versions-does-fabric-support}
 
 Oficialmente, o Fabric é compatível com todas as versões do Minecraft começando da snapshot `18w43b` em diante, e versões da `1.14` em diante.
 
-### Onde eu posso baixar mods publicados do Fabric?
+## Onde eu posso baixar mods publicados do Fabric? {#where-can-i-download-published-fabric-mods}
 
 :::info
 Você deve sempre verificar se os mods são de fontes confiáveis. Visite o guia [Encontrando Mods Confiáveis](./finding-mods) para mais informações.
 :::
 
-A maioria dos autores publicam seus mods no [Modrinth](https://modrinth.com/mods?g=categories:%27fabric%27) e no [CurseForge](https://www.curseforge.com/minecraft/search?class=mc-mods&gameVersionTypeId=4), no entando, alguns preferem publicá-los em seus sites pessoais, ou em outras plataformas, como um repositório GitHub.
+A maioria dos autores publicam seus mods no [Modrinth](https://modrinth.com/mods?g=categories:%27fabric%27) e no [CurseForge](https://www.curseforge.com/minecraft/search?class=mc-mods\&gameVersionTypeId=4), no entando, alguns preferem publicá-los em seus sites pessoais, ou em outras plataformas, como um repositório GitHub.
```

Comment:
We should do something about this: Crowdin escapes Markdown characters in links, but this breaks the links.

@modmuss50 @IMB11

Reply:
This isn't actually crowdin, its people either using external software to edit the pages or they're using a translator to translate strings. Google Translate has a tendency to escape link characters.